src/lib/resources/folder.resource.ts

```diff
@@ -45,4 +45,12 @@ export class FolderResource extends BaseResource {
 
     return this.getVoFromResponse<FolderResponse>(response, 'FolderVO');
   }
+
+  public async getAppFolders(): Promise<FolderVO[]> {
```

Comment:
This returns a (promise of) a list of folders; is it ever the case that there will be more than one "Apps" folder? If not, could we rename this to `getAppsFolder` and return the single element in the list?

Reply:
To capture some conversation: this was originally returning an array of FolderVOs because the backend claims to do so, but we saw that the backend API in fact just returns the first "Apps" folder it finds in "folder/getAppRoot," so doing so here seems perfectly appropriate.